docs/user-guide.md

```diff
@@ -2223,6 +2223,22 @@ instance is created.</td>
 <td>Run with the Ansible debugging flag enabled.  With debugging
 enabled, passwords appear in logfiles.</td>
 </tr>
+<tr>
+<td></td>
+<td><p><pre>
+--install_workload_agent
+</pre></p></td>
+<td></td>
+<td>Whether to install <a href="https://github.com/GoogleCloudPlatform/workloadagent">Google Cloud Agent for Compute Workloads</a></td>
```

Comment:
Better link: https://cloud.google.com/compute/docs/instances/agent-for-compute-workloads